src/v2/Apps/Categories/__tests__/CategoriesApp.jest.tsx

```diff
@@ -30,4 +30,31 @@ describe("CategoriesApp", () => {
     renderWithRelay()
     expect(screen.getByText("The Art Genome Project")).toBeInTheDocument()
   })
+
+  it("displays families and genes", () => {
+    renderWithRelay({
+      GeneFamilyConnection: () => ({
+        edges: [
+          {
+            node: {
+              name: "Styles and Movements",
+              genes: [
+                {
+                  displayName: "Early Randomcore",
+                },
+              ],
+            },
+          },
+        ],
+      }),
+    })
+
+    expect(
+      screen.getByRole("heading", { name: "Styles and Movements" })
+    ).toBeInTheDocument()
```

Comment:
My first move was to try the familiar query

```js
screen.getByText("Styles and Movements")
```

…but when I did that I got this helpful error message:

![multi](https://user-images.githubusercontent.com/140521/134744677-40a25a15-2ad0-4af1-bea1-4a7acf2d63c7.png)


…Thus realizing that I was matching both the horizontal pill nav as well as the subheading for the "Styles and Movements" family.

I was only concerned with the subheading, which led me to write the query this way…

```js
screen.getByRole("heading", { name: "Styles and Movements" })
```

…thus leaning into the [ARIA roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#roles) that are kind of the [bread and butter](https://testing-library.com/docs/queries/about/#priority) of working with RTL. 

<img src="https://user-images.githubusercontent.com/140521/134744533-90b41fed-3fef-47da-b6a6-56b72c91c19e.png" width="400" />


**_That_** assertion in turn led me to change the markup in the component from a vanilla `div` to a semantic `h2` [up there](https://github.com/artsy/force/pull/8484/commits/23e4ec89d164fd2425f8e5f1f19b8750f68c6388#diff-cf581cd1b36174c1baf0e997d9501fe5fe6feb104500a86cc5a15252e0af9c72R30) 

A nice little illustration of how RTL thinking can actually lead us to write more a11y-friendly components.

Thank you for coming to my ted talk